src/tests/end2end/BufferTests.cpp

```diff
@@ -209,7 +209,7 @@ TEST_P(BufferSetSubDataTests, SmallDataAtOffset) {
 
     constexpr uint32_t kOffset = 2000;
     uint32_t value = 298371;
-    buffer.SetSubData(kOffset / 4, 1, &value);
+    buffer.SetSubData(kOffset, sizeof(value), &value);
```

Comment:
Can we also have 1 byte setsubdata tests and at an offset that's not aligned?

Reply:
Done.